modules/compliance-must-gather.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/co-support.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="compliance-must-gather_{context}"]
+= Using the must-gather tool for the Compliance Operator
+
+Starting in Compliance Operator v1.6.0, you can collect data about the Compliance Operator resources by running the `must-gather` command with the Compliance Operator image.
+
+[NOTE]
+====
+Consider using `must-gather` when opening support cases or filing bug reports, as it provides additional details about the Operator configuration and logs.
+====
+
+.Procedure
+
+* Run the following command to collect data about the Compliance Operator: 
++
+[source,terminal]
+----
+$ oc adm must-gather --image=$(oc get csv compliance-operator.v1.6.0 -o=jsonpath='{.spec.relatedImages[?(@.name=="must-gather")].image}')

```

Comment:
This command actually doesn't work because of bug https://issues.redhat.com/browse/OCPBUGS-42185